SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-4028/SpatialAuthorityTest.cpp

```diff
@@ -0,0 +1,359 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
We're trying to move away from bug ids as folder names, please give this a descriptive name instead.